src/containers/costume-tab.jsx

```diff
@@ -18,6 +18,7 @@ import {
 } from '../reducers/modals';
 
 import addBlankCostumeIcon from '../components/asset-panel/icon--add-blank-costume.svg';
+import addLibraryBackdropIcon from '../components/asset-panel/icon--add-backdrop-lib.svg';
```

Comment:
Maybe we should put shared icons in a shared place so we don't have to duplicate them?